layers/state_tracker.cpp

```diff
@@ -4234,14 +4263,14 @@ void ValidationStateTracker::PreCallRecordCmdPushDescriptorSetWithTemplateKHR(Vk
 
     cb_state->RecordCmd(CMD_PUSHDESCRIPTORSETWITHTEMPLATEKHR);
     auto template_state = Get<UPDATE_TEMPLATE_STATE>(descriptorUpdateTemplate);
-    if (template_state) {
-        auto layout_data = Get<PIPELINE_LAYOUT_STATE>(layout);
-        auto dsl = layout_data ? layout_data->GetDsl(set) : nullptr;
+    auto layout_data = Get<PIPELINE_LAYOUT_STATE>(layout);
+    if (template_state && layout_data) {
```

Comment:
Avoid code indentation for readability.

```suggestion
    if (template_state == nullptr || layout_data == nullptr) { return; }
```